src/EnergyPlus/HeatBalanceAirManager.cc

```diff
@@ -286,7 +286,7 @@ void GetSimpleAirModelInputs(EnergyPlusData &state, bool &ErrorsFound) // IF err
 
     // Following used for reporting
     state.dataHeatBal->ZnAirRpt.allocate(state.dataGlobal->NumOfZones);
-    if (state.dataHeatBal->doSpaceHeatBalanceSimulation) {
+    if (state.dataHeatBal->doSpaceHeatBalance) {
```

Comment:
Hmm, this is wrong, should be `doSpaceHeatBalanceSimulation || doSpaceHeatBalanceSizing`. It might be ok, but not safe if sizing is false and simulation is true. Will fix.